lua/gk.lua

```diff
@@ -20,10 +20,19 @@ return function (net_conf, numa_table)
 	gk_conf.max_num_ipv6_rules = 1024
 	gk_conf.num_ipv6_tbl8s = 65536
 
- 	-- TODO Edit of the FIB table.
+	gk_conf.max_num_ipv6_neighbors = 65536
+
+	local front_net_prefix = "10.0.0.1/24"
+	local front_net_prefix6 = "2001:db8::1/32"
+
+	local back_net_prefix = "10.0.1.1/24"
+	local back_net_prefix6 = "2002:db8::1/32"
```

Comment:
Shouldn't the GK block be able to get the front and back network prefixes from the network configuration?